torch/csrc/jit/mobile/register_mobile_ops.cpp

```diff
@@ -23,5 +23,138 @@ static auto registry0 = torch::RegisterOperators().op(
   [](at::Tensor a, at::Scalar b, at::Scalar c) ->at::Tensor {
     return at::add(a, b, c);
   })
+).op(
```

Comment:
These changes seem like a different PR

Reply:
They will be moved to a new PR. This PR would only include the operators that are needed in unit test.